constraint/pkg/client/rego_helpers.go

```diff
@@ -98,3 +98,66 @@ func checkDataAccess(module *ast.Module) Errors {
 	}
 	return nil
 }
+
+// rule name -> arity
+type ruleArities map[string]int
+
+// requireRules makes sure the listed rules are specified with the required arity
+func requireRules(rego string, reqs ruleArities) error {
+	module, err := ast.ParseModule("foo", rego)
```

Comment:
`"foo"` will show up in error messages. We'd be better off using something a bit more informative...some ideas:

"lib"
"hooks"
"built-in"

Reply:
done